src/utils/wallet.js

```diff
@@ -277,11 +281,28 @@ export class Wallet {
       return sendJson('POST', `${ACCOUNT_HELPER_URL}/account/${phoneNumber}/${accountId}/requestCode`)
    }
 
-   async validateCode(phoneNumber, accountId, securityCode) {
-      const key = this.key_store.getKey(accountId)
-      const signer = new nearlib.SimpleKeyStoreSigner(this.key_store);
-      const { signature } = key ? signer.signBuffer(Buffer.from(securityCode), accountId) : undefined;
-      return sendJson('POST', `${ACCOUNT_HELPER_URL}/account/${phoneNumber}/${accountId}/validateCode`, { securityCode, signature })
+   validateCode(phoneNumber, accountId, postData) {
+      return sendJson('POST', `${ACCOUNT_HELPER_URL}/account/${phoneNumber}/${accountId}/validateCode`, postData)
+   }
+
+   async setupAccountRecovery(phoneNumber, accountId, securityCode) {
+      // TODO: Don't hardcode key
+      const HELPER_KEY = '22skMptHjFWNyuEWY22ftn2AbLPSYpmYwGJRGwpNHbTV';
```

Comment:
is this supposed to be a secret? are we going to change it for production, since this was already out in the public?

Reply:
This isn't supposed to be a secret, it's public key, but the intent is to pass it via environment variable (same as for contract helper)

Reply:
Done